packages/web/src/lib/logger.ts

```diff
@@ -0,0 +1,52 @@
+declare const debug: debug.Debug & { debug: debug.Debug; default: debug.Debug };
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace debug {
```

Comment:
I am not aware for naming conventions of namespaces. Should it start from small case (debug)?